### ISSUE TITLE
Add man page for V2

### ISSUE THREAD

**Issue by ceuk** [aws/aws-cli#5097]
`man aws` should be equivalent to `aws help`, `aws x help` etc

**Comment by KaibaLopez** [comment]
Hi @ceuk,
I'm not sure if "should" or "fail" is the correct wording here, man is a linux command that helps describe other linux commands not a general application descriptor, so maybe this would be a feature request we could consider but not really a bug.

Can you describe a situation where having this would allow you to do something that can't be done with the `aws help` command is it just command preference?
I think it would be nice to add this to the cli so I'll mark it as a feature request for now, it just might be low priority if it's just a preference thing.

**Comment by ceuk** [comment]
Hi @KaibaLopez  you're absolutely right, apologies for my wording, I was on my mobile on desktop mode because I had just had to jump up to deal with a crying baby (the joys of quarantine). So I was a bit briefer than I might have been and I was bit stressed at the time. Thanks for the gracious reply to a rather terse message. 

In answer to your question, from a design perspective (and admittedly, a Linux perspective),  `man aws` is more predictable than `aws help`. I actually tried: `man aws` and `aws --help` before finding `aws help`. The fact that it's `aws --version` made it slightly more confusing for me to reason out which of the possible syntaxes would be correct. 

Man pages also tend to be fairly standardised which makes them easy for those used to them to parse.

Finally (and I feel like I'm stretching here) they conform to the "everything is a file" Unix philosophy. Not sure what practical benefit that has in this instance though :)

Perhaps these might come across as fairly flimsy justifications, but in terms of cost-benefit, it seems like it would maybe be a low bar.

Thanks for the consideration
